### DISCUSSION TITLE
How can I read changes for specific version

### DISCUSSION THREAD

**Discussion by l-bujakowski** [JetBrains/ideavim#856] [Q&A] [answered]
Hello. I see a new version of IdeaVim is available for my IDE: 2.10.2.
I update, and I'm curious about what changes were made. I check the "What's new" tab, and see only a link to [Changelog](https://youtrack.jetbrains.com/issues/VIM?q=State:%20Fixed%20Fix%20versions:%202.10.2).

It's empty. I try to approach it from the github side - i open this repo, read the description, go to the 'changelog' chapter.

"Since version 2.9.0, the changelog can be found on YouTrack"

I go to [youtrack](https://youtrack.jetbrains.com/issues/VIM?q=State:%20Fixed%20sort%20by:%20updated%20) and I have no idea how to filter out only for example changes from 2.9.0, or only for 2.10.2. I see the tags are meshed up from all jetbrains projects.

I was very easy to read the changelog until 2.9.0. All changes for given version at a glance. How Can I get similar infirmation on youtrack?

**Comment by AlexPl292** [comment, marked as answer]
Hi there. You can see changes in the specific version by specifying the "fixed in" field like [this](https://youtrack.jetbrains.com/issues/VIM?q=State:%20Fixed%20sort%20by:%20updated%20fixed%20in:%202.10.0).

Sorry for the inconvenience, we've faced a few issues with existing changelog automation after updating the release process. So we have to turn it off and rewrite it at some moment.

**Reply by OptimusCrime** [reply]
Thanks for answering this question. I was wondering about the same thing. 

I am not sure if there's anything that can be done with the Youtrack setup, but it is _super_ confusing and _super_ slow, at least to me. Trying to navigate anything related to IdeaVim is borderline impossible. Have you considerd moving issues for this plugin to GitHub instead?

**Reply by l-bujakowski** [reply]
Thank you for response. It was helpful.
